apps/nextjs/next.config.ts

```diff
@@ -32,15 +32,31 @@ const nextConfig: NextConfig = {
   experimental: {
     optimizePackageImports: ["@mantine/core", "@mantine/hooks", "@tabler/icons-react"],
     turbopackFileSystemCacheForDev: true,
+    // Reduce memory usage by limiting concurrent requests
+    serverActions: {
+      bodySizeLimit: "1mb",
+    },
+    // Reduce memory usage by limiting worker threads
+    workerThreads: false,
+    // Optimize memory usage
+    memoryBasedWorkersCount: true,
   },
+  // Reduce memory usage in production
+  compress: true,
+  poweredByHeader: false,
   transpilePackages: ["@homarr/ui", "@homarr/notifications", "@homarr/modals", "@homarr/spotlight", "@homarr/widgets"],
+  // Reduce memory by limiting image optimization
   images: {
     localPatterns: [
       {
         pathname: "/**",
         search: "",
       },
     ],
+    minimumCacheTTL: 60,
```

Comment:
Remove, does not improve memory usage, see https://nextjs.org/docs/app/api-reference/components/image#minimumcachettl